rfdetr/main.py

```diff
@@ -151,6 +159,17 @@ def train(self, callbacks: DefaultDict[str, List[Callable]], **kwargs):
         print(args)
         device = torch.device(args.device)
 
+        # Initialize early stopping if enabled
+        if args.early_stopping:
```

Comment:
I think we should place initialization of early stopping next to `MetricsPlotSink` and `MetricsTensorBoardSink`, in `rfdetr/detr.py` to keep things consistant. 

```python
metrics_plot_sink = MetricsPlotSink(output_dir=config.output_dir)
self.callbacks["on_fit_epoch_end"].append(metrics_plot_sink.update)
self.callbacks["on_train_end"].append(metrics_plot_sink.save)

metrics_tensor_board_sink = MetricsTensorBoardSink(output_dir=config.output_dir)
self.callbacks["on_fit_epoch_end"].append(metrics_tensor_board_sink.update)
self.callbacks["on_train_end"].append(metrics_tensor_board_sink.close)
```

@Matvezy is there any reason why we couldn't do that?